components/kyma-environment-broker/cmd/broker/main.go

```diff
@@ -184,9 +186,26 @@ func main() {
 	bundleBuilder := ias.NewBundleBuilder(httpClient, cfg.IAS)
 	iasTypeSetter := provisioning.NewIASType(bundleBuilder, cfg.IAS.Disabled)
 
+	// metrics collectors
```

Comment:
IMO the main is going to be bigger and bigger. Can we just hide all those 16 lines of code to 1 ine? Sth like

```go

eventPublisher := metrics.RegisterAll()

// then in use that in managers, e.g.
provisionManager := provisioning.NewManager(db.Operations(), eventPublisher, logs.WithField("provisioning", "manager"))
	
```

IMO defining all those metrics gives us more mess instead of readability

Reply:
refactored